### PR TITLE
feat(Forms): deprecate Form.useError in favor of `Form.useValidation` featuring `setFieldStatus` to handle the status (error) of a single field

### PR DESCRIPTION
**Pull request by tujoworker**

### CODE REVIEW

File: packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/getting-started.mdx [hunk 1]
```diff
@@ -359,7 +359,7 @@ In addition, you can add your own validation to a field component. This is done
 
 Fields which have the `disabled` prop or the `readOnly` prop, will skip validation.
 
-For monitoring and set your form errors, you can use the [useError](/uilib/extensions/forms/Form/useError) hook.
+For monitoring and set your form errors, you can use the [useValidation](/uilib/extensions/forms/Form/useValidation) hook.
```
Comment (joakbjerk) [inline packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/getting-started.mdx:362]:
```suggestion
For monitoring and setting your form errors, you can use the [useValidation](/uilib/extensions/forms/Form/useValidation) hook.
```

File: packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/Form/useValidation/info.mdx [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+        })
+      }}
+    >
+      <Field.String path="/path/to/field" />
+    </Form.Handler>
+  )
+}
+```
+
+The remove the field status, you can use `setFieldStatus('/path/to/field', { error: null })`.
```
Comment (langz) [inline packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/Form/useValidation/info.mdx:125]:
```suggestion
To remove the field status, you can use `setFieldStatus('/path/to/field', { error: null })`.
```

**Review by langz — APPROVED** [review]

**Review by joakbjerk — APPROVED** [review]
Looks good from testing locally

**Pull request merged**